src/Particular.PlatformSample.ServiceControl/Particular.PlatformSample.ServiceControl.csproj

```
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <IncludeBuildOutput>False</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>True</SuppressDependenciesWhenPacking>
+    <Description>Particular ServiceControl binaries for use by Particular.PlatformSample. Not intended for use outside of Particular.PlatformSample.</Description>
+    <PackageProjectUrl>https://docs.particular.net/servicecontrol/</PackageProjectUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\ServiceControl.Transports.Learning\ServiceControl.Transports.Learning.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\ServiceControl\ServiceControl.csproj" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Particular.Packaging" Version="0.2.1" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="bin\debug\net461\*.*" Exclude="**\*.config;**\*.pdb" PackagePath="platform\servicecontrol\servicecontrol-instance" Visible="false" Link="hidden"/>
```

This needs a tweak:

```suggestion
    <Content Include="$(OutputPath)**\*.*" Exclude="**\*.config;**\*.pdb" PackagePath="platform\servicecontrol\servicecontrol-instance" Visible="false" Link="hidden"/>
```

However, this isn't actually good enough. This is evaluated too early, so it will produce an empty package if you don't already have build output. It would also mean that it could be using an outdated list of files (if something changed between builds.)

This means it's going to need to go into the output folders of the ServiceControl and ServiceControl.Transports.Learning projects instead. When the packaging project is evaluated, those two will already have been built, so everything should work.

---

```suggestion
    <Content Include="..\ServiceControl\$(OutputPath)**\*.*" Exclude="**\*.config;**\*.pdb" PackagePath="platform\servicecontrol\servicecontrol-instance" Visible="false" Link="hidden"/>
    <Content Include="..\ServiceControl.Transports.Learning\$(OutputPath)**\*.*" Exclude="**\*.config;**\*.pdb" PackagePath="platform\servicecontrol\servicecontrol-instance" Visible="false" Link="hidden"/>
```

Should do the trick.